style.css

```diff
@@ -1,141 +1,148 @@
-#navbar{
-    background-color: rgb(145, 32, 32);
-    color: rgb(255, 255, 255);
-    overflow: hidden;
-    position: fixed;
-    top: 0;
-    width: 100%;
-
-
+#navbar {
+  background-color: rgb(145, 32, 32);
+  color: rgb(255, 255, 255);
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  width: 100%;
 }
-#navbar, ul{
-    list-style: none;
-  
 
+#navbar,
+ul {
+  list-style: none;
 }
+
 /* #navbar, ul, li{
-   
-} */
-#navbar, ul, li, a{
-    color: rgb(255, 255, 255);
-    text-decoration: none;
-    padding: 2px 8px;
-    float: right;
-    text-align: center;
-    font-size: 16px;
 
+} */
+#navbar,
+ul,
+li,
+a {
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  padding: 2px 8px;
+  float: right;
+  text-align: center;
+  font-size: 16px;
 }
 
-
- 
-
-#about{
-    background-color: rgb(16, 16, 44);
-    padding-top: 50px;
-   
+#about {
+  background-color: rgb(16, 16, 44);
+  padding-top: 50px;
 }
-#name{
-    text-align: center;
-    color: rgb(255, 255, 255);
-    text-decoration: none;
 
+#name {
+  text-align: center;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
 }
-#about, h3{
-    text-align: center;
-    color: rgb(172, 41, 41);
-}
-
 
-#projects{
-    background-color: rgb(16, 16, 44);
-    
-}
-#projects, h1{
-    text-align: center;
-    color: rgb(255, 255, 255);
-    text-decoration: underline;
+#about,
+h3 {
+  text-align: center;
+  color: rgb(172, 41, 41);
 }
 
-.projects{
-    display: flex;
-    height: px;
-    border: 1px solid #444;
-    padding: 10px;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    align-items: stretch;
-    align-content: stretch;
-}
-.flex{
-    width: 300px;
-    background-color: black;
-    margin: 10px;
-    color: rgb(255, 255, 255);
-    font-size: 150%;
-    text-align: center;
-}
-.project1{
-    order: 1;
-}
-.project2{
-    order: 2;
-    
+#projects {
+  background-color: rgb(16, 16, 44);
 }
-.project3{
```

Comment:
Please use grid template and flex-box to position the cards and images correctly.

Reply:
Good job here 👍